docs/rl-algorithms/c51.md

```diff
@@ -180,6 +181,129 @@ Tracked experiments and game play videos:
+
+* Uses [Jax](https://github.com/google/jax), [Flax](https://github.com/google/flax), and [Optax](https://github.com/deepmind/optax) instead of `torch`.  [c51_atari_jax.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/c51_atari_jax.py) is roughly 25% faster than  [c51_atari.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/c51_atari.py)
+* For playing Atari games. It uses convolutional layers and common atari-based pre-processing techniques.
+* Works with the Atari's pixel `Box` observation space of shape `(210, 160, 3)`
+* Works with the `Discrete` action space
+
+### Usage
+
+```bash
+poetry install --with atari,jax
+poetry run pip install --upgrade "jax[cuda]==0.3.17" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
+python cleanrl/c51_atari_jax.py --env-id BreakoutNoFrameskip-v4
+python cleanrl/c51_atari_jax.py --env-id PongNoFrameskip-v4
+```
+
+
+### Explanation of the logged metrics
+See [related docs](/rl-algorithms/c51/#explanation-of-the-logged-metrics) for `c51_atari.py`.
+
+
+### Implementation details
+See [related docs](/rl-algorithms/c51/#implementation-details) for `c51_atari.py`.
+
+
+### Experiment results
+
+To run benchmark experiments, see :material-github: [benchmark/c51.sh](https://github.com/vwxyzjn/cleanrl/blob/master/benchmark/c51.sh). Specifically, execute the following command:
+
+<script src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2Fvwxyzjn%2Fcleanrl%2Fblob%2Fmaster%2Fbenchmark%2Fc51.sh%23L23-L29&style=github&showBorder=on&showLineNumbers=on&showFileMeta=on&showCopy=on"></script>
+
+Below are the average episodic returns for `c51_atari_jax.py`. 
+
+
+| Environment             | `c51_atari_jax.py` 10M steps | `c51_atari.py` 10M steps | (Bellemare et al., 2017, Figure 14)[^1] 50M steps | (Hessel et al., 2017, Figure 5)[^3]   |
+| ----------------------- | ---------------------------- | ------------------------ | ------------------------------------------------- | ------------------------------------- |
+| BreakoutNoFrameskip-v4  | 448.56 ± 17.02               | 461.86 ± 69.65           | 748                                               | ~500 at 10M steps, ~600 at 50M steps  |
+| PongNoFrameskip-v4      | 19.88 ± 0.31                 | 19.46 ± 0.70             | 20.9                                              | ~20 10M steps, ~20 at 50M steps       |
+| BeamRiderNoFrameskip-v4 | 9504.91 ± 709.69             | 9592.90 ± 2270.15        | 14,074                                            | ~12000 10M steps, ~14000 at 50M steps |
+
+
+
+Learning curves:
+<div class="grid-container">
+<img src="../c51/jax/BeamRiderNoFrameskip-v4.png">
+<img src="../c51/jax/BeamRiderNoFrameskip-v4-time.png">
+
+<img src="../c51/jax/BreakoutNoFrameskip-v4.png">
+<img src="../c51/jax/BreakoutNoFrameskip-v4-time.png">
+
+<img src="../c51/jax/PongNoFrameskip-v4.png">
+<img src="../c51/jax/PongNoFrameskip-v4-time.png">
+</div>
```

Comment:
These look great! A minor note for the future: we shouldn't feel obligated to export the wandb curves manually in the future anymore -- the rlops utility should give us the `compare.png` and `compare-time.png` that we can use directly to save manual labor :)